### PR TITLE
Members and Communities Counters

### PR DESCRIPTION
**Pull request by royriver23**
## Purpose

Adding new queries to keep track of communities and members' counts. See: https://app.dework.xyz/indiedao/lvl-protocol?taskId=07a1f130-d5a8-4f99-9eaf-309af54eb5a4

## How to test

- You can go into Fauna Dashboard and run the queries yourself from `util/queries.js` 
- Additionally, you may want to actually create members from the app to make sure the counter is updated.

## How to backfill counters in development and production?

* Setup production key locally or ask for someone who could :)
* Run: 
 ```bash
node scripts/migrations/1656853059410-sync-members-communities-count.js
```

### CODE REVIEW

File: util/api/fauna.js [hunk 1]
```diff
@@ -93,9 +112,10 @@ export const updateMemberCache = async ({ id, nftSrc, ens }) => {
   return resp.member
 }
 
-export const getAccessListFirst480 = async () => {
+export const getAccessList = async (size = 480) => {
```
Comment (mckelvey) [inline util/api/fauna.js:115]: I like this refactor away from the explicit number... nice!

File: fauna/data/initialize_communities_counter.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const CommunityInitialCounter = {
+  collection: 'CommunitiesCounter',
+  index: 'communities_counter',
+  key: 'counter',
+  data: [
+    {
+      counter: 0,
```
Comment (mckelvey) [inline fauna/data/initialize_communities_counter.js:7]: Wondering, do we need to initialize these with the right count? Or am I missing how we’d make the state of the counters match the existing number of members/communities in the DB at present?

**Review by mckelvey — COMMENTED** [review]
@royriver23 — Looks great! I just had a question as to making the numbers match the state of the DB...

File: fauna/data/initialize_communities_counter.js (hunk 2 above)
Comment (mckelvey) [reply fauna/data/initialize_communities_counter.js:7]: OH... this is `TBD` in the main comment... seems like we should figure it out... possible to do as a migration?

Comment (royriver23) [reply fauna/data/initialize_communities_counter.js:7]: Done, I just added the migration along with instructions on how to run it :)

Comment (mckelvey) [reply fauna/data/initialize_communities_counter.js:7]: Coolness!

**Review by mckelvey — APPROVED** [review]
Awesome @royriver23 — let’s connect with @nicovalencia to get keys to run this correctly when merged to develop and main. 💯

**Pull request merged**